.gitattributes

```diff
@@ -2,8 +2,11 @@
 /.git export-ignore
 /.github export-ignore
 /.wordpress-org export-ignore
+/assets export-ignore
```

Comment:
✅ Plugin repo [guideline four (readable code) met](https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/#4-code-must-be-mostly-human-readable) by inclusion of the link to the GitHub repo.